src/Hotel.java

```diff
@@ -5,24 +5,29 @@
     private String name;
     private ArrayList<Room> rooms;
 
-    Hotel(String name, ArrayList<Room> rooms) {
+    Hotel(String name, ArrayList<Room> rooms)
+    {
         this.name = name;
         this.rooms = rooms;
     }
 
-    String getName(){
+    String getName()
+    {
         return this.name;
     }
 
-    ArrayList<Room> getRooms(){
+    ArrayList<Room> getRooms()
+    {
         return this.rooms;
     }
 
-    int hotelNumberOfRooms(){
+    int hotelNumberOfRooms()
```

Comment:
If this is a getter method, rename the method using the convention. 

- getRoomCount or something like this.

also access modifier and "this."